### PR TITLE
Makefile: install recvtty and make install depend on build

### PR DESCRIPTION
**Pull request by metux**
Signed-off-by: Enrico Weigelt, metux IT consult <info@metux.net>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -102,8 +102,9 @@ shell: runcimage
 		-v $(CURDIR):/go/src/$(PROJECT) \
 		$(RUNC_IMAGE) bash
 
-install:
+install: runc recvtty
 	install -D -m0755 runc $(DESTDIR)$(BINDIR)/runc
+	install -D -m0755 contrib/cmd/recvtty/recvtty $(DESTDIR)$(BINDIR)/recvtty
```
Comment (kolyshkin) [inline Makefile:107]: Not sure if we should install `recvtty`. It is not meant to be a standalone utility, but rather an example of how to write something like that.

**Comment by cyphar** [comment]
NACK, `recvtty` is not a real utility people should use, it's a PoC so people can implement `--console-socket` handling in their own container managers.

**Comment by metux** [comment]
So, higher layers (eg. moby) don't use it all ? I've just seen that the precompiled docker-ce deb's ship that file, so I added it.

**Comment by cyphar** [comment]
No none of the higher layers use it at all (Docker/containerd use containerd-shim, and cri-o/podman use conmon), I'm a bit surprised that docker-ce would ship it.

**Comment by metux** [comment]
> No none of the higher layers use it at all (Docker/containerd use containerd-shim, and cri-o/podman use conmon), I'm a bit surprised that docker-ce would ship it.

At least the one I've found in Debian.

But if it's not neded at all, I probably won't need to package it.

**Pull request closed without merging**